spec/models/article_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+
+  context "when a title has a length of at least five characters" do
+    it "is valid" do
+      article = Article.new(
+        title: "Title",
+        category: Category.new(),
+        text: "I'm an article"
+      )
+      expect(article).to be_valid
```

Comment:
Nitpick - I'd probably add a new line before the `expect`. (Same for other test cases)